src/components/revurdering/oppsummering/oppsummeringsblokk/Oppsummeringsblokk.tsx

```diff
@@ -43,6 +49,17 @@ const Intro = (props: { revurdering: Revurdering }) => {
                         tittel: intl.formatMessage({ id: 'label.begrunnelse' }),
                         verdi: props.revurdering.begrunnelse,
                     },
+                    {
+                        tittel: intl.formatMessage({ id: 'label.attestant' }),
+                        verdi: pipe(
+                            props.revurdering.attesteringer.filter((a) => a.underkjennelse === null),
+                            maxBy(Ord.contramap((a: Attestering) => a.opprettet)(S.Ord)),
+                            Option.fold(
+                                () => '–',
+                                (a) => a.attestant
```

Comment:
Det skal väl kun max finnes 1 attestering som er "godkjent". Isåfall hade det nog hållt med bara `A.find( a => a.underkjennelse === null), Option.fold(...)`

Reply:
Okei, hvis vi kan stole på at det er tilfellet så forenkler det jo litt for frontenden sin del.

Reply:
Etter diskusjon på Slack kom vi til at vi beholder `maxBy`-opplegget for sikkerhets skyld.